src/style/colors.js

```diff
@@ -6,6 +6,7 @@ export const orange = '#ffa834';
 export const lightGray = '#d2dae2';
 export const almostWhite = '#f0f4f7';
 export const darkWhite = '#b3b3ba';
+export const softGray = '#8A8A8A';
```

Comment:
I don't remember this color. Please do not add new colors that don't belong to the mocks.